### ISSUE TITLE
Problem with wars and alliances

### ISSUE THREAD

**Issue by Xanaa** [vawser/warcraft_universalis#77]
1. I have opened this Issue becouse its not sure if we will create one big alliance (all with all) or some smaller allied alliances (multiple central alliances and suballiances) or only one leader
(i.e. Stormwind allied with Darnassus and Ironforge but Ironforge not with Darnassus).
What do you thing?

2. If we decide to create war, this war will be only with leaders and closest alliances or this will be total war (all allied on all allied)?

**Comment by Xanaa** [comment]
If someone will ask me, the best will be option ' all allied with all ', but this will be terribly big file

**Comment by Blanchard33** [comment]
For local wars I suggest only the involved parties, like the Hinterlands War in cataclysm would be Revantusk and Forsaken vs. Wildhammer and Highvale elves. 

For the alliances all allied with all could work, but perhaps if we give Horde / Alliance members country modifiers "Horde Member" or  "Alliance Member" than create an event that fires if a country with the "Horde Member" modifier declares war with an "Alliance Member" country and vice versa than a decision to call in other "Alliance Member" countries fires, which in turn drags the other Horde countries into the war.

**Comment by Xanaa** [comment]
I'm not sure if I understand correctly. Country modifiers will decide if ie. Darnassus go to war along side with Stormwind without "Alliance" system?

**Comment by Blanchard33** [comment]
The country modifiers enable the AI to enact decisions to which will result in all Alliance countries in the same subcontinent allying with the same faction, ensuring the proper alliances are maintained throughout the game. The system I made is dynamic and will fire shortly after the game starts. Check it out for yourself in the decisions folder and test it out in game. Its not completely finished though but so far it works well and is promising. The old alliances we worked on are archived in the archive folder, so don't worry I didn't delete that work.
